deployments/arbitrumsepolia_addresses.json

```diff
@@ -2,6 +2,14 @@
   "name": "arbitrumsepolia",
   "chainId": "421614",
   "addresses": {
-    "AccessControlManager": "0xa36AD96441cB931D8dFEAAaC97D3FaB4B39E590F"
+    "AccessControlManager": "0xa36AD96441cB931D8dFEAAaC97D3FaB4B39E590F",
+    "CriticalTimelock": "0xC7d1b39482beaf7b9ac76Ac613c4872A97949121",
+    "DefaultProxyAdmin": "0xD61A0D1165d1f1c1ADD12c66f0994fCFeAA66A2e",
```

Comment:
I would reuse the proxy admin deployed in the oracle repo: `0xA78A1Df376c3CEeBC5Fab574fe6EdDbbF76fd03e`

Reply:
[a676ab9](https://github.com/VenusProtocol/governance-contracts/pull/21/commits/a676ab9bd0b04791aaf67d6a4d647af5e3eb8973)